lib/constants/dimensions.dart

```diff
@@ -35,7 +35,7 @@ class Dimensions {
   static const double appBarHeight = 32.0;
   static const double minimumInteractionWidgetLength = 48.0;
   static const double recommendedMentorTileWidth = 112.0;
-  static const Size upcomingSessionTile = Size(184.0, 96.0);
+  static const Size upcomingSessionTile = Size(80.0, 150.0);
```

Comment:
Not super important at this point, but when defining sizes always try to choose a number that is a multiple of 8 preferably, or by 4 in cases you need more precision (e.g. smaller elements). This is to give the illusion of all elements being neatly organized in a grid.

Check out this for more details: https://m2.material.io/design/layout/spacing-methods.html#baseline-grid

Reply:
Ah great tips! Changed the constants. Thanks!